examples/hacker_news_assets/hacker_news_assets/utils/table.py

```diff
@@ -0,0 +1,77 @@
+import textwrap
+from typing import Any, Callable, Mapping, NamedTuple, Optional, Set
+
+from dagster import OpDefinition
+from dagster.core.asset_defs.asset_in import AssetIn
+from dagster.core.asset_defs.decorators import _Asset
+
+
+class Column(NamedTuple):
+    type_name: str
+    description: str
+
+
+def table(
```

Comment:
should this be part of a different diff?  for one, not sure about how we would want to represent columns if wanted to start doing column-level lineage as well...

Reply:
Yeah, that's reasonable.  Removing it.